pkg/cmd/cli/cmd/newapp.go

```diff
@@ -258,8 +258,21 @@ func RunNewApplication(fullName string, f *clientcmd.Factory, out io.Writer, c *
 				installing = append(installing, t)
 			}
 		case *buildapi.BuildConfig:
-			if len(t.Spec.Triggers) > 0 {
+			triggered := false
+			for _, trigger := range t.Spec.Triggers {
+				switch trigger.Type {
+				case buildapi.ImageChangeBuildTriggerType:
+					triggered = true
+					break
+				case buildapi.ConfigChangeBuildTriggerType:
+					triggered = true
+					break
+				}
+			}
+			if triggered {
 				fmt.Fprintf(out, "%sBuild scheduled for %q - use the logs command to track its progress.\n", indent, t.Name)
+			} else {
+				fmt.Fprintf(out, "%sBuildConfig %q does not include any automatic triggers, use start-build to start a build.\n", indent, t.Name)
```

Comment:
I did BuildConfig because that's what the rest of the output shows, eg:

`    BuildConfig "ruby-sample-build" created`

but ok, changed to Build config.